db.js

```diff
@@ -0,0 +1,27 @@
+const pg = require ('pg')
+
+const client = new pg.Client('postgres://localhost/books_db')
+
+const syncAndSeed = async () => {
+    const SQL = `
+        DROP TABLE IF EXISTS Book;
+        CREATE TABLE Book (
```

Comment:
i think even if you created the table as `Book` (capital B), it'll still get created in sql as `book` (lowercase b)
same with lines 13-19- sql automatically casts it to all lowercase
to actually make a table called `Book`, you'd need quotes- `create table "Book"`
but no one really makes capital letter tables in sql- just too much confusion LOL